### PR TITLE
Rr/bugfix z offset

### LINKED ISSUE #42: Z tool offset not ported to master
M218 does not have Z tool offset on by default

### PR DESCRIPTION
**Pull request by riningerr**
This fix adds Z tool offset to M218. It wasn't ported over in the rebase.

fixes #42

dogerabbit

### CODE REVIEW

**Comment by kdumontnu** [comment]
Looks swell :octopus:

File: tests/test_main.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
     direct_write=True,
     direct_write_mode='serial',
     #printer_port="/dev/tty.usbmodem1411",
-    printer_port="COM8",
+    printer_port="COM14",
```
Comment (kevingelion) [inline tests/test_main.py]: We discussed making the COM port a command line argument. I can do this tomorrow.

**Comment by kdumontnu** [comment]
"I watched Ricky type it out" -Jack

**Comment by riningerr** [comment]
"Looks good to me!", said @jminardi

**Pull request merged**